### PR TITLE
[FIX] create+fund account before subscribing

### PR DESCRIPTION
**Pull request by sentientwaffle**
If the account doesn't exist yet, it can't be subscribed to, so autofund first.

This is necessary to fix the visualization. /cc @MatthewPhinney

### CODE REVIEW

File: lib/ledgers/five-bells-ledger.js [hunk 1]
```diff
@@ -87,22 +88,22 @@ FiveBellsLedger.prototype.subscribe = function * (target_uri) {
...
-      body: {
-        name: this.credentials.username,
-        balance: '1500000',
-        connector: config.server.base_uri,
-        password: this.credentials.password
-      }
-    }, 'could not create account at ledger ' + this.id)
-  }
+FiveBellsLedger.prototype._autofund = function * () {
+  log.info('creating account at ' + this.id)
```
Comment (clark800) [inline lib/ledgers/five-bells-ledger.js]: Is it possible that the account already exists?

Comment (sentientwaffle) [inline lib/ledgers/five-bells-ledger.js]: Yes, but it doesn't matter, this is just a debug flag.

Comment (clark800) [inline lib/ledgers/five-bells-ledger.js]: It could cause confusion for the person debugging though

**Comment by sentientwaffle** [comment]
@clark800 ok, I changed the log line

**Comment by clark800** [comment]
LGTM

**Pull request merged**